### PR TITLE
MAINT-27892: Check labels for gamification pages

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**
**Problem** : Check labels for gamification pages

### CODE REVIEW

File: portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue [hunk 1]
```diff
@@ -140,7 +140,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
                       v-model="editedbadge.enabled"
                       :disabled="editedbadge.domainDTO==null||!editedbadge.domainDTO.enabled"
                       type="checkbox">
-                    <span class="slider round"></span>
+                    <span class="slider round"><span class="absolute-yes">{{ $t(`exoplatform.gamification.YES`,"YES") }}</span></span>
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue]: Is there a need to add an extra SPAN element here ?

File: portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue [hunk 2]
```diff
@@ -264,10 +264,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
                 </div>
               </td>
               <td class="badge-title-col">
-                <div>{{ $t(`badge.title.${badge.title}`,badge.title) }}</div>
+                <div>{{ $t(`badge.title.${badge.title.replace(' ','')}`,badge.title) }}</div>
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue:267]: Why is this needed ? could you do a String.trim() on the badge object itself.

File: portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue [hunk 3]
```diff
@@ -557,7 +557,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         position: absolute;
         left: -20px;
         z-index: 1;
-        content: "YES";
+        content: "";
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue:560]: Is the content rule still needed ?

File: portlets/src/main/frontend/src/apps/components/domain/DomainList.vue [hunk 4]
```diff
@@ -660,7 +663,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         position: absolute;
         left: -20px;
         z-index: 1;
-        content: "YES";
+        content: "";
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/components/domain/DomainList.vue:666]: Same as above

File: portlets/src/main/frontend/src/apps/components/domain/SaveDomainForm.vue [hunk 5]
```diff
@@ -69,15 +69,15 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+                  :placeholder="$t('exoplatform.gamification.gamificationinformation.add.domain.description')">
                             </textarea>
               </form>
 
               <form>
                 <label class="col-form-label pt-0">{{ $t(`exoplatform.gamification.enabled`,"Enabled") }}:</label>
                 <label class="uiSwitchBtn">
                   <input v-model="domain.enabled" type="checkbox">
-                  <span class="slider round"></span>
+                  <span class="slider round"><span class="absolute-yes">{{ $t(`exoplatform.gamification.YES`,"YES") }}</span></span>
```
Comment (ahamdi) [inline portlets/src/main/frontend/src/apps/components/domain/SaveDomainForm.vue]: same as above

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue (hunk 1 above)
Comment (Rdidmohamedelhadi) [reply portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue]: Yes, we must add it because we need to apply specific css to this span.

File: portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue (hunk 2 above)
Comment (Rdidmohamedelhadi) [reply portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue:267]: `String.trim()` remove  only space from beginning and end,but we need to remove spaces from the middle

File: portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue (hunk 3 above)
Comment (Rdidmohamedelhadi) [reply portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue:560]: yes , when we remove it the content will not displayed

File: portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue (hunk 1 above)
Comment (ahamdi) [reply portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue]: Couldn't we merge both SPAN elements?

Comment (Rdidmohamedelhadi) [reply portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue]: We cannot merge two spans because each span has a specific css

**Review by ahamdi — APPROVED** [review]
LGTM
